src/Resources/config/content.xml

```diff
@@ -28,7 +28,7 @@
             </call>
 
             <call method="setCkEditorConfig">
-                <argument>%cmf_sonata_phpcr_admin_integration.content.ivory_ckeditor%</argument>
+                <argument>%cmf_sonata_phpcr_admin_integration.ivory_ckeditor.config%</argument>
```

Comment:
i would prefer that, yes. because we would always want the specific config, and we don't know if we ever will have more than one. (maybe for editing a block type that can contain html...)